wdn/templates_5.0/js-src/events-band.js

```diff
@@ -1,7 +1,7 @@
 define([
 	'wdn',
 	'jquery',
-	'moment',
+  'plugins/moment-timezone-with-data',
```

Comment:
tabs/spaces (an aside: we need a massive whitespace cleanup and reset on this project)